NASA-AMMOS/aerie-ui
# Feat/1088/activity directive creator

Closes #1088

Testing:
1. Create a plan
2. Create an activity
3. Double click on the activity to view it in the Selected Activity panel
4. Ensure the forms displays a "Created By" field
5. Login as a different user and modify the activity start time (as admin)
6. Ensure the activity "Created By" field remains the same and that the "Last Modified By" field reflects the new user.
7. Ensure "Created By" and "Last Modified By" are available as Activity Directive Table column options.
